src/odemis/acq/fastem_conf.py

```diff
@@ -45,8 +45,11 @@
         # the beam here.
         "blanker": False,
         "immersion": False,  # disable to get a larger field of view
-        "horizontalFoV": 1.5e-3,
-        "resolution": (1024, 884),  # px
+        "horizontalFoV": 1.5e-3,  # maximum FoV without seeing the pole-piece (with T1, immersion off).
+        # XT usually uses a rectangular ratio for the resolution such as 1536 x 1024. Thus, the maximum width is
+        # reached earlier, but the heights could be in principle still increased. By using a more square aspect ratio,
+        # it is possible to increase the area per tile and thus reduce the number of tiles that need to be acquired.
+        "resolution": (1024, 884),  # [px]
```

Comment:
@pieleric

Reply:
The keypoint here, which is maybe not very clear is that the **FoV is fixed** at its maximum value. So increasing the number of pixels scanned means that you'll get a better resolution. We don't aim at a specific number of pixel total, but a specific area. Hence, squarer is better, because that means a larger area in height.

Reply:
Then I think this comment can be clarified by adding "for a fixed FoV' in the second sentence, and "physically scanned area" in the last:

```
# XT usually uses a rectangular ratio for the resolution such as 1536 x 1024. Thus, for a fixed FoV the maximum width is
# reached earlier, but the heights could be in principle still increased. By using a more square aspect ratio,
# it is possible to increase the physically scanned area per tile and thus reduce the number of tiles that need to be acquired.
```

Reply:
I updated it.